docs/kb/error-codes-messages.md

```diff
@@ -1615,6 +1627,42 @@ requests.
 
 **Solution**: Resubmit the request using a valid [authentication](/#authentication) method.
 
+## [](#error-code-128)Error Code: 128
```

Comment:
I see that Error Code 127 is API_ERROR_INVALID_SHARD so I'm fine with it being omitted.